### DISCUSSION TITLE
Can I change the appearance of the selected shape?

### DISCUSSION THREAD

**Discussion by kobe651jp** [excalidraw/excalidraw#5998] [Q&A] [answered]
The shape of the selected object on the official site is looks nice.

Is it possible to reproduce the same?

![image](https://user-images.githubusercontent.com/44263972/208049479-63713ecd-e321-4fa4-92c9-950905caf317.png)

**Comment by ad1992** [comment, marked as answer]
@kobe651jp we will soon release the package with the design changes which will have the same appearance as well as in [excalidraw](https://excalidraw.com)
